updatecli/updatecli.d/updatecanal.yaml

```diff
@@ -0,0 +1,87 @@
+---
+name: "Update Canal version" 
+
+sources:
+  flannel:
+    name: Get flannel version
+    kind: githubrelease
+    spec:
+      owner: rancher
+      repository: image-build-flannel
+      token: '{{ requiredEnv "UPDATECLI_GITHUB_TOKEN" }}'
```

Comment:
Try using this instead https://github.com/rancher/image-build-cni-plugins/blob/main/updatecli/updatecli.d/updateplugins.yaml#L11